src/config.ts

```diff
@@ -34,25 +34,24 @@ export const SIDEBAR = {
 		{ text: 'Deploy', link: 'en/guides/deploy' },
 		{ text: 'Environment Variables', link: 'en/guides/environment-variables' },
 		{ text: 'Import Aliases', link: 'en/guides/aliases' },
+		{ text: 'Using Integrations', link: 'en/guides/integrations-guide' },
 		// MOVE "Pagination": Into "Routing Rules" Reference
 		{ text: 'Pagination', link: 'en/guides/pagination' },
-		// ADD: Integrations
 		{ text: 'RSS', link: 'en/guides/rss' },
 		// MOVE "Manual Setup": Into "Installation" as some sort of other tab
 		//          Similar to Tailwind? https://tailwindcss.com/docs/installation
 		{ text: 'Manual Setup', link: 'en/guides/manual-setup' },
```

Comment:
Yes, I think Manual Setup can be right on the installation page. I think this is better.  :)